tests/yast2_gui/yast2_storage_ng.pm

```diff
@@ -0,0 +1,242 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST2 ...
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "y2x11test";
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_tumbleweed is_opensuse);
+
+sub add_logical_volume {
+    my ($lvname, $role) = shift;
+    sleep 1;
```

Comment:
try to avoid such unconditional sleeps , especially when you doing this just before wait_screen_change. I think you can just remove it or play with wait_screen_change params to make it do this wait

Reply:
Ok, I'll try something else :+1: